extensions/stats/plugin.cc

```diff
@@ -587,8 +587,15 @@ bool PluginRootContext::report(::Wasm::Common::RequestInfo& request_info,
   for (size_t i = 0; i < expressions_.size(); i++) {
     if (!evaluateExpression(expressions_[i],
                             &istio_dimensions_.at(count_standard_labels + i))) {
-      LOG_TRACE(absl::StrCat("Failed to evaluate expression at slot: " +
-                             std::to_string(i)));
+      std::string expression_str;
+      for (const auto& expression : input_expressions_) {
+        if (expression.second == i) {
+          expression_str = expression.first;
+          break;
+        }
+      }
+      LOG_TRACE(absl::StrCat("Failed to evaluate expression: <", expression_str,
+                             "> at slot: ", std::to_string(i)));
       istio_dimensions_[count_standard_labels + i] = "";
```

Comment:
i'm beginning to wonder if `""` is the best value here. didn't we have something elsewhere in this that was auto-converting `""` to `"unknown"` ? are we now inconsistent in missing/errors for dimensions?

Reply:
do we want to change this to unknown or unmapped?

Reply:
`unknown` sounds better to me. It will be a good indicator that config takes effect, but evaluation fails.